.github/workflows/main.yml

```diff
@@ -14,3 +17,17 @@ jobs:
         fetch-depth: 0
     - name: Is Rebased on master?
       uses: cyberark/enforce-rebase@v2
+
+  commit_style:
+    runs-on: ubuntu-latest
+    name: Enforce commit style
+
+    steps:
+    - name: Checkout the repo
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Check commit style
+      uses: telday/enforce-commit-style@v1
```

Comment:
do we want to move this to the `cyberark` org?

Reply:
oh i see that `https://github.com/telday/enforce-commit-style` redirects to `https://github.com/cyberark/enforce-commit-style`. in that case can we use the `cyberark` one?

Reply:
My bad, also going to need to fix that on the main repo